### PR TITLE
Fix dynamo.export(aten=True) for condition op

### PR DESCRIPTION
**Pull request by SherlockNoMad**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #92361

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

**Review by voznesenskym — APPROVED** [review]

File: functorch/experimental/_cond.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ class UnsupportedAliasMutationException(RuntimeError):
 
 
 def trace_cond(proxy_mode, func_overload, pred, true_fn, false_fn, operands):
-    assert isinstance(operands, list), "Cond operands must be a list of tensors"
+    assert isinstance(operands, (list, tuple)), "Cond operands must be a list or tuple of tensors"
```
Comment (SherlockNoMad) [inline functorch/experimental/_cond.py:40]: operands is passed in as a tuple, instead of a list

**Pull request closed without merging**